aiida/cmdline/commands/cmd_devel.py

```diff
@@ -58,6 +59,28 @@ def get_valid_test_paths():
     return valid_test_paths
 
 
+@verdi_devel.command('check-load-time')
+def devel_check_load_time():
+    """Check for common indicators that slowdown `verdi`.
+
+    Currently known pathways that will slow down the loading time:
+
+        * the database environment is loaded
+        * the `aiida.orm` module has been imported
+
+    If either of these conditions are true, the command will raise a critical error
+    """
+    from aiida.manage.manager import get_manager
+
+    manager = get_manager()
+
+    if manager.backend_loaded:
+        echo.echo_critical('potential `verdi` speed problem: database backend is loaded.')
+
+    if 'aiida.orm' in sys.modules:
+        echo.echo_critical('potential `verdi` speed problem: `aiida.orm` module is imported.')
+
```

Comment:
Turned it into a `echo_success`

Reply:
That's great - somehow I don't see it in the diff. Is it there or did it get lost somewhere along the line?

Reply:
I haven't pushed the changes yet, because I will anyway have to rebase once #3394 is merged, but is waiting a second approval after I addressed Gio's comments.